### ISSUE TITLE
Don't back to summary report after edit transaction in report 

### ISSUE THREAD

**Issue by dennys** [moneymanagerex/moneymanagerex#6235]
How to reproduce
1. Choose a report, e.g. Where the money goes
2. Click a category and sub-category
3. Click an ID to open the 'edit transaction' window
4. Close the window
5. It backs to the summary report.

Is it possible to keep it in the summary report? 
Because when I want to modify several transaction, it's more convenient, thanks.

**Comment by whalley** [comment]
Don't think this is a quick fix. After a change we can easily leave the summary report open, but if values are changed the only mechanism we currently have is to re-generate the report.

**Comment by dennys** [comment]
I totally agree this is not easy to fix... and thanks for your contribution to this project. If you have a test version, I can help to test it, thanks.
